woocommerce/woocommerce-gateway-payfast
# Bump WooCommerce "tested up to" 8.3

### All Submissions:


---

### Changes proposed in this Pull Request:

- Bump WooCommerce "tested up to" version 8.3.
- Bump WooCommerce minimum supported version to 8.1.
- Bump WordPress minimum supported version to 6.2.

Closes #157 

### Steps to test the changes in this Pull Request:

1. Run the e2e test using GitHub action to test this PR with WC 8.3
2. All tests should be passed. 
3. Manual Test - Plugin should give Notice when using an unsupported version of WooCommerce. 

### Changelog entry

> Dev - Bump WooCommerce "tested up to" version 8.3.
> Dev - Bump WooCommerce minimum supported version to 8.1.
> Dev - Bump WordPress minimum supported version to 6.2.
